app/renderer/ui/components/main/sections/smartContracts/tabs/history/index.tsx

```diff
@@ -52,30 +52,29 @@ class TabHistory extends TabComponent<any & Props> {
             </CardDefault>
           </Wrapper>
         </div>
-
         <div className={`${styles.right}`}>
           <p className={styles.title}>About Smart Contracts</p>
           <p className={styles.text}>
-            Smart contracts are programs that live in the Witnet blockchain. They have their own
-            addresses for receiving Witnet tokens, but the ability to spend those tokens is
-            determined by the code of the program itself.
+                Smart contracts are programs that live in the Witnet blockchain. They have their own
```

Comment:
I think this indentation should be fixed ;)

Reply:
done